MMUL-BASH-01/Exercises/Day-2/Exercise.007.solutions

```diff
@@ -0,0 +1,41 @@
+1) Supposing your user is "mmul" and your machine is "machine":
+
+> ssh mmul@machine
+
+2) Query the status of the service:
+
+sudo systemctl status mmul_exercise.service
+
+"● mmul_exercise.service - Example
+   Loaded: loaded (/etc/systemd/system/mmul_exercise.service; disabled; vendor preset: disabled)
+   Active: active (running) since Tue 2019-12-03 11:07:20 CET; 4s ago
+ Main PID: 2415 (bash)
+   CGroup: /system.slice/mmul_exercise.service
+           ├─2415 /bin/bash /home/mmul/service_action
+           └─2417 sleep 5
+
+Dec 03 11:07:20 machine systemd[1]: Started Example.
+Dec 03 11:07:20 machine bash[2415]: Example service from author: mmul"
+
+The service is "disabled".
+
+3) Enable the service with systemctl:
+
+sudo systemctl enable mmul_exercise.service
+
+"Created symlink from /etc/systemd/system/multi-user.target.wants/mmul_exercise.service to /etc/systemd/system/mmul_exercise.service."
+
+4) Query the status again. Notiche the "enabled" bit:
+
+sudo systemctl status mmul_exercise.service
```

Comment:
Please change using $(whoami) instead of mmul

Reply:
replaced username with $(whoami) command